challenges/objects-arrays.js

```diff
@@ -89,6 +134,13 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+
+function sciandanimalname(zooAnimals){
```

Comment:
forEach should look like this `
zooAnimals.forEach(function(animal) {
  animalNames.push`(`Name: ${animal.animal_name}, Scientific: ${animal.scientific_name}.`);`
}); `